events/wc_fel_events.txt

```diff
@@ -827,7 +829,12 @@ character_event = {
 				trait = being_demon
 				trait = fair trait = proud trait = patient trait = deceitful trait = arbitrary trait = cynical trait = cruel trait = envious
 			}
-			new_character = { narrative_event = { id = WCFEL.1005 } }
+			new_character = {
+				set_character_flag = is_kiljaeden
+				add_artifact = legion_doomhorn
+				add_artifact = scepter_of_sargeras
```

Comment:
Not mentioned this change in the changelog.

Reply:
fixed